sigsci/assets/monitors/excessiveblockedHTTP.json

```
@@ -0,0 +1,29 @@
+{
+	"id": 20344850,
+	"name": "SigSci - excessive blocked http requests",
+	"type": "query alert",
+	"query": "avg(last_4h):anomalies(sum:sigsci.agent.waf.block{*}.as_count(), 'agile', 2, direction='above', alert_window='last_5m', interval=60, count_default_zero='true', seasonality='daily') >= 1",
+	"message": "Signal Sciences Web Application Firewall has been blocking an unusually high number of web requests. This may be indicative of a volumetric attack against one or more of your web properties or API endpoints. Details are available in your Signal Sciences console at https://dashboard.signalsciences.net/.",
+	"tags": [],
+	"options": {
+		"notify_audit": false,
+		"locked": false,
+		"timeout_h": 0,
+		"silenced": {},
+		"include_tags": true,
+		"no_data_timeframe": null,
+		"require_full_window": true,
+		"new_host_delay": 300,
+		"notify_no_data": false,
+		"renotify_interval": 0,
+		"escalation_message": "",
+		"threshold_windows": {
+			"recovery_window": "last_15m",
+			"trigger_window": "last_5m"
+		},
+		"thresholds": {
+			"critical": 1,
+			"critical_recovery": 0
+		}
+	}
```

```suggestion
	},
	"recommended_monitor_metadata": {
		"description": "Notify your team when Signal Sciences Web Application Firewall has been blocking an unusually high number of web requests."
	}
```

Let's include this field as well